meson.build

```diff
@@ -134,19 +134,26 @@ deps += dependency('icu-i18n', version: '>=4.8.1.1')
 dep_avail = []
 foreach dep: [
     # audio, in order of precedence
-    ['libpulse', '', 'PulseAudio'],
-    ['alsa', '', 'ALSA'],
-    ['portaudio-2.0', '', 'PortAudio'],
-    ['openal', '>=0.0.8', 'OpenAL'],
+    ['libpulse', '', 'PulseAudio', []],
+    ['alsa', '', 'ALSA', []],
+    ['portaudio-2.0', '', 'PortAudio', []],
+    ['openal', '>=0.0.8', 'OpenAL', []],
     # video
-    ['ffms2', '', 'FFMS2'],
+    ['ffms2', '', 'FFMS2', ['ffms2', 'ffms2_dep']],
     # other
-    ['fftw3', '', 'FFTW3'],
-    ['hunspell', '', 'Hunspell'], # needs a proper port
-    ['uchardet', '', 'uchardet'], # needs a proper port
+    ['fftw3', '', 'FFTW3', []],
+    ['hunspell', '', 'Hunspell', []], # needs a proper port
+    ['uchardet', '', 'uchardet', []], # needs a proper port
 ]
-    d = dependency(dep[0], version: dep[1] != '' ? dep[1]: '>=0',
-                   required: false)
+    # [provide] section is ignored if required is false;
+    # must provided define fallback explicitly
+    # (with meson 0.56 you can do allow_fallback: true)
```

Comment:
Can you add a commented snippet with what this needs to be replaced with once meson 0.56 releases? It should be out soonish I think.

Reply:
Done.